rs.py

```diff
@@ -44,4 +44,4 @@ def classify():
 
 if __name__ == '__main__':
     configure_service('resources/config/app_config.cfg')
```

Comment:
What are the contents of the `app_config.cfg` file? To make it easier to use with Docker, we should define a default config or support configuration through environment files.